homework/tests/example.spec.js

```diff
@@ -17,7 +17,7 @@ const data = [
 
```

Comment:
Not all operations have tests. E.g. Subtract operation is not covered. Please add respective test.

Reply:
Please update test file name to reflect what is being tested. E.g. CalculatorPageTests.spec.js